### ISSUE TITLE
Always show "accessbility services must be disabled" on Hyper OS 2.0

### ISSUE THREAD

**Issue by xbbsky** [osudroid/osu-droid#466]
### Issue description

- Expected result: What were you expecting to happen?

Expect a way to play the game.

- Actual result: What actually happened?

![Screenshot_2024-12-11-23-08-32-349_ru.nsu.ccfit.zuev.osuplus.jpg](https://github.com/user-attachments/assets/a4b18467-c323-4f96-9e05-bc399d9a1c20)

Always show the alert, but I have disabled all the accessbility settings I can find.

### Steps to reproduce the issue
1. Just open the game on my Xiaomi 14 Ultra which runs Hyper OS 2.0

#### Additional details / Screenshot / Clip
- ...

### Versions
* Android version: 15.0
* Game version: 1.8-P4

**Comment by iqionly** [comment]
I face this issue before, the message dissapear after i disable accessibility of app, (says autoclicker) and then restart the game. If you try disable but not restart the game, this message not disappear.

**Comment by Cierra-Runis** [comment]
> Read the dialog

It's system component... We player can't disable it without rooting.

**Comment by NeJex-Ppl** [comment]
yep, after some kind of update in hos 2.0, this also happened on xiaomi 15 pro :(
